charts/consul/values.yaml

```diff
@@ -2330,7 +2330,7 @@ connectInject:
         cpu: "50m"
       limits:
         memory: "150Mi"
-        cpu: "50m"
```

Comment:
We could also entertain setting the `limits.cpu` value to `null` as with the sidecar above. Then it will land on a box with 50m and use as much as it can to start.

Did you want to bump the `requests.cpu` value as well?

Reply:
I could change `limits.cpu` to null and bump up `requests.cpu` to `250m` to be conservative. CPU increase for init-container here seems to be temporary so ok to set limits to `null`